templates/psp.yaml

```diff
@@ -33,7 +33,7 @@ subjects:
     name: envproxy
     namespace: {{ .Release.Namespace | quote }}
   - kind: ServiceAccount
-    name: environments
+    name: coder
```

Comment:
apologies if I'm missing something, just for my edification here:

1. what's envproxy using this service account for?
2. in a coderd world, are we going to use this coder account, or are we going to use `cemanager`?
3. are we going to have a `coder` service account as well as `coderd` (formerly `cemanager`) service account?